packages/grid/x-data-grid-pro/src/tests/filtering.DataGridPro.test.tsx

```diff
@@ -75,6 +75,53 @@ describe('<DataGridPro /> - Filter', () => {
     expect(getColumnValues(0)).to.deep.equal(['Adidas', 'Puma']);
   });
 
+  it('should not apply items that are incomplet with AND operator', () => {
+    render(
+      <TestCase
+        filterModel={{
+          items: [
```

Comment:
```suggestion
          linkOperator: GridLinkOperator.And,
          items: [
```
To not rely on default link operator